config/config.go

```diff
@@ -49,19 +49,29 @@ func (g Generator) ParamString() string {
 }
 
 func (g Generator) ParamStringWithOut() string {
+	// If no out path was specified, use the package directory.
 	outPath := g.OutPath()
+	if outPath == "" {
+		outPath = g.ConfigDir
+	}
 	params := g.ParamString()
 	if params == "" {
 		return outPath
 	}
 	return params + ":" + outPath
 }
 
+// Determine an out path for a generator to write generated files to.
+// An empty string is returned if no 'out' key was set.
 func (g Generator) OutPath() string {
-	if g.Out != "" {
+	if g.Out == "" {
+		return ""
+	}
+	if filepath.IsAbs(g.Out) {
 		return g.Out
+	} else {
```

Comment:
you can drop this else and unindent the line below

Reply:
Done.